.travis.yml

```diff
@@ -82,9 +82,16 @@ jobs:
         - ./node_modules/.bin/eslint app/assets/src --ext .js,.jsx
         - npm test
     - name: "Docker Build"
-      language: shell
+      language: python
```

Comment:
was this supposed to change?

Reply:
I had changed it thinking we needed python for "pip install". It turns out that travis CI includes python with the "minimal/shell" build anyways. I will change it back and see what happens.